great_expectations/rule_based_profiler/parameter_container.py

```diff
@@ -274,20 +269,24 @@ def build_parameter_container_for_variables(
     Returns:
         ParameterContainer containing all variables
     """
-    variable_config_key: str
-    variable_config_value: Any
-    parameter_values: Dict[str, Any] = {}
-    for variable_config_key, variable_config_value in variables_configs.items():
-        variable_config_key = f"{VARIABLES_KEY}{variable_config_key}"
-        parameter_values[variable_config_key] = variable_config_value
-
-    parameter_container = ParameterContainer(parameter_nodes=None)
-    build_parameter_container(
-        parameter_container=parameter_container, parameter_values=parameter_values
+    parameter_node = ParameterNode()
+    key: str
+    value: Any
+    for key, value in variables_configs.items():
+        key_parts: List[str] = [PARAMETER_NAME_ROOT_FOR_VARIABLES] + key.split(
+            FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER
+        )
```

Comment:
I totally forgot you could do this with lists 😄 (add them that is)

Reply:
I actually timed both and the `+` was faster. Maybe it can set aside the space upfront and doesn't temporarily make the initial list?

Reply:
Yeah, I'm going to start using this more.

Reply:
Interesting... I have been avoiding the "+" on lists because I was under the impression that allocating a new list would take more time.  Good find, @billdirks !  Thanks!